subsys/logging/sys_log_net.c

```diff
@@ -0,0 +1,224 @@
+/*
+ * Copyright (c) 2018 Intel Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <logging/sys_log.h>
+
+#include <net/net_pkt.h>
+#include <net/net_context.h>
+
+/* Set this to 1 if you want to see what is being sent to server */
+#define DEBUG_PRINTING 0
+
+static struct net_context *ctx;
+static struct sockaddr server_addr;
+
+/* As there is no way to figure out these values in the hook
+ * function, use some pre-defined values.
+ */
+static const int facility = 16; /* local0 */
+static const int severity = 6; /* info */
```

Comment:
Note that with the planned LOG_* API (which will replace SYS_LOG) you won't need to do this kind of hard-coding, since you'll be getting the logging level as proper meta-data (with SYS_LOG it's just embedded as a prefix in the string)

Reply:
Good that this issue with needed func parameters in being planned in the future.